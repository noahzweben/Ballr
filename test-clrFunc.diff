1c1,5
<   ./test-clrFunc failed on ./test-clrFunc
---
> 20
> 250
> 100
> SDL could not initialize! SDL Error: No available video device
> Window initialization failed.
\ No newline at end of file
