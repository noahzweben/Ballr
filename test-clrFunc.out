  ./test-clrFunc failed on ./test-clrFunc
