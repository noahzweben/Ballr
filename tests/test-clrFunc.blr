func color my_clr_func(){
	
	int r = 20;
	int g = 250;
	int b = 100;
	return (r, g, b);
}

gameboard g1{
	clr = (251,142,74);
	size = (680,420);
	init -> {

		color my_clr = my_clr_func();
		print(my_clr[0]);
		print(my_clr[1]);
		print(my_clr[2]);
        
  	}
}