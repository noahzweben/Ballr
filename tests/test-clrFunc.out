20
250
100
SDL could not initialize! SDL Error: No available video device
Window initialization failed.